client/my-sites/domains/controller.jsx

```diff
@@ -30,8 +30,7 @@ import { getCurrentUser } from 'state/current-user/selectors';
  */
 const productsList = productsFactory();
 
-module.exports = {
-
+export default {
```

Comment:
How about we export all those functions individually (or via a group at the end of the file)? The current approach introduces an unnecessary indentation :/ (I know, I know, nitpicking, but since we're already refactoring...)